_vcsh_bash

```diff
@@ -52,11 +52,15 @@ _vcsh_git_command () {
 }
 
 _vcsh () {
-	local cur prev words cword OPTS
+	local cur prev words OPTS
 	_init_completion -n = || return
 
-	local repos cmds
-	repos=( $(command vcsh list) )
+	local r reponames
+	local -A repos
+	mapfile -t reponames < <(command vcsh list)
+	for r in "${reponames[@]}"; do repos["$r"]="$r"; done
+	unset r reponames
+	local cmds
```

Comment:
In `bash` can you declare a variable as local and define it in the same line? If so lets do that, but mostly I'm asking out of curiosity. On ZSH I'm accustomed to this being one operation.

Reply:
I'm just used to [SC2155](https://github.com/koalaman/shellcheck/wiki/SC2155) and now declare and assign separately. In this case since it is a static list it is not required to be separate. I can make it a single line declare and assign if you like.

Reply:
Thanks for the docs pointer here, that makes sense I guess. Carry on!